cert_manager/certbot.py

```diff
@@ -34,7 +35,15 @@ def request_cert(self, domains):
         for domain in domains:
             command += ["--domain", domain]
         result = subprocess.run(command, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        result.check_returncode()
+        try:
+            result.check_returncode()
+        except subprocess.CalledProcessError as e:
+            if self.email_failure_alert:
+                message = "Subject: {subject}\n\n{stderr}".format(
+                    subject="Alert: certbot failed to issue certificate", stderr=e.stderr.decode("utf-8")
+                )
+                send_email(self.contact_email, message)
```

Comment:
@ahmedbilal, I don't think it is a good idea to use the value provided for the `--contact-email` option without a way to override it. That email address is used for registering with Let's Encrypt and is only monitored once a day or so. In production, we may want to alert a different email address, like `urgent`, when these errors happen.